### ISSUE TITLE
Update to checkra1n 0.12.0 beta

### ISSUE THREAD

**Issue by c-hri-s** [foxlet/bootra1n#112]
A new release of checkra1n has been released, please update bootra1n with this version: 0.12.0 beta

Thanks

**Comment by Yikerdoodle** [comment]
Yes, sooner than later would be greatly appreciated, since it adds full support for A10 devices and partial support for A11.

**Comment by oooceees** [comment]
Good thing that someone talked about it

**Comment by ankeongonthe** [comment]
agree

**Comment by Zalienated** [comment]
Also waiting for an update before jailbreak attempt...

**Comment by jwamp14** [comment]
Also waiting!

**Comment by Goofychex** [comment]
:)

**Comment by Goofychex** [comment]
Please update ASAP! I cannot wait to Jailbreak my 6s. 

- Thanks!!

**Comment by romulus2** [comment]
I updated the device to ios 14.2, when I'm on bootra1n 0.11.0 does not recognize the port, I believe it's on account of the version, does anyone confirm this doubt for me? Please..

**Comment by Yikerdoodle** [comment]
@romulus2 iOS 14.2 isn’t supported on that version of BootRa1n. That is probably why it isn’t detected.

**Comment by romulus2** [comment]
I just saw that maybe it is not the version of bootra1n 0.11.0 it is also not a problem on my cable, the port just doesn't work, I even saw the USB functions in the setup, I will do it on a different laptop .... could someone help me  ?

**Comment by c-hri-s** [comment]
@romulus2 it's 100% the version of checkra1n included with bootra1n - it doesn't support what you're trying to do.

tbh at this stage I'd suggest https://github.com/asineth0/checkn1x as this seems pretty dead.

**Comment by Zalienated** [comment]
> 
> 
> 
> tbh at this stage I'd suggest https://github.com/asineth0/checkn1x as this seems pretty dead.

Another alternative is https://github.com/raspberryenvoie/odysseyn1x

**Comment by NeuroMancer108** [comment]
> > tbh at this stage I'd suggest https://github.com/asineth0/checkn1x as this seems pretty dead.
> 
> Another alternative is https://github.com/raspberryenvoie/odysseyn1x

**Awesome tool bro! Thank you for sharing**

**Comment by oooceees** [comment]
does it work?

**Comment by oooceees** [comment]
> @romulus2 it's 100% the version of checkra1n included with bootra1n - it doesn't support what you're trying to do.
> 
> tbh at this stage I'd suggest https://github.com/asineth0/checkn1x as this seems pretty dead.

you sure it does support iOS14.2 ?

**Comment by oooceees** [comment]
> @romulus2 it's 100% the version of checkra1n included with bootra1n - it doesn't support what you're trying to do.
> 
> tbh at this stage I'd suggest https://github.com/asineth0/checkn1x as this seems pretty dead.

You sure it does support iOS14.2 on iPhone 6s ?

**Comment by c-hri-s** [comment]
It has the latest checkra1n.
You can determine if that's going to do what you need here: https://checkra.in/news/2020/09/iOS-14-announcement

(yes, in other words)

**Comment by romulus2** [comment]
someone recommends me a bypass for ios 0.14.2???... Please

**Comment by ankeongonthe** [comment]
Another tool : Jailm8

**Comment by romulus2** [comment]
does it circumvent the activation lock?

**Comment by c-hri-s** [comment]
No, it won't activate your stolen phone. Do your own research.

**Comment by romulus2** [comment]
> No, it won't activate your stolen phone. Do your own research.

sorry for the research, in case the phone was lost is 5 months stopped I left up to the chip in the hope of someone calling me, just for information effect

**Comment by oooceees** [comment]
Is there any resource for (PortraitXI-latest version) tweak for free ?

**Comment by Yikerdoodle** [comment]
So in my personal experience, the fastest most reliable way to always have the latest version of checkra1n is to dual boot Ubuntu and Windows with around 30GB of space for the entire Ubuntu system and follow the installation instructions at http://checkra.in/linux after setting up Ubuntu. 
Another option would be to use a live USB with Ubuntu that has persistent storage on it so that it stores files across reboots (this way you don't have to install checkra1n every time you boot to it in BIOS).

**Comment by cnrkuo** [comment]
https://github.com/foxlet/bootra1n/issues/127#issuecomment-753865100
